assets/python/ml/config.yaml

```diff
@@ -0,0 +1,29 @@
+doc_boilerplate:
```

Comment:
The variable and file names could be more explanatory

Reply:
Renamed to `documentation_box_common_text`

`config.yaml` is a configuration file that, in later tasks planned for flavor generation, will contain additional configuration for the project